index.html

```diff
@@ -66,6 +66,199 @@ <h3>I'm a Software Developer</h3>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+        
+       
+      </div>
+     
+    </section>
+    <section class="about-myself">
+      <div class="about-me">
+        <h1 class="about-me-heading">About me</h1>
+        <p class="about-me-note">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <span class="get-my-resume">Get my resume</span>
+      </div>
```

Comment:
- [ ]  Kindly make sure the button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=SU8lUrhODQH8Lc5d-0)) as this is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) of the project 

> Implement the button interactions (enable, hover, etc.).

<kbd>
<img width="296" alt="Screenshot 2023-04-30 at 2 05 08 AM" src="https://user-images.githubusercontent.com/44798044/235324255-52f77572-9e89-4363-b38f-30ebc88a6ba6.png">
</kbd>